### DISCUSSION TITLE
Meet and Greet

### DISCUSSION THREAD

**Discussion by arafattehsin** [AzureAiDevs/hub#8] [General]
Hi folks! 👋

This thread is all about you. Tell us a few things about you:

- Where are you joining from?
- What are you doing these days with AI?
- What is your favorite Azure AI offering? 

I'd go first. I'm [Arafat Tehsin](https://github.com/arafattehsin), Microsoft MVP for Artificial Intelligence. I am based in Sydney, Australia. I work with Applied AI quite extensively and my favorite part of Azure AI is Cognitive Services.

**Comment by dev123in432** [comment]
Hello,

My name is David, and I'm an Azure cloud engineer and dotnet web app developer based in Sydney. I came across this community while searching for help with creating a Blazor web app using Azure OpenAI. The April 30 days of Azure-AI is exactly what I need.

I enjoy learning and sharing with others through experimentation and have had great fun playing with the Azure ChatGPT playground, creating silly bots. It's amazing to see the different settings and prompts that come out. However, my wife has banned me from talking about chatGPT, AI, or DALL-E because I talk about it too much. 

I'm interested in creating basic Blazor web apps on top of Azure OpenAI, search and language services to create demos that I can share with others and learn about using my companies own IP and data in these models.

**Reply by arafattehsin** [reply]
Wow! Another one from Sydney.

Hi @dev123in432 👋 so good to know about you and your interests around Azure Open AI offerings. Blazor sounds cool.

It's hard to stop talking to ChatGPT once you get on with it...

**Reply by dev123in432** [reply]
Yep, very addictive ;)

**Reply by gloveboxes** [reply]
Hey David (and Arafat), I'm in the AI Advocacy team and I've had the fun of building out this site.

The site is now live, and delighted you've found useful, and yes my family getting bored of me going on about ChatGPT too😊 .   Like you both, I'm a Sydneysider😀 

Let me know in the Call for Content discussion if there is anything you'd like featured on the site. 

Cheers Dave

**Reply by dev123in432** [reply]
thanks Dave, will let you know

**Reply by ghadirim** [reply]
hello, joining from Brooklyn, NY. I'm a Microsoft Security MVP. I'm interested in security use cases (Sentinel/Defender for Cloud/SOC investigation, triage and response use cases.

**Reply by arafattehsin** [reply]
👋 @ghadirim - nice to meet you, Mona. Cybersecurity has always been a cool thing for me. Now, I know who to reach out in this case :)

**Comment by keenthinker** [comment]
Hi,

nice to meet you all. 👋

I'm Pavel, an Azure and .NET enthusiast from Europe. I'm not writing software in my day job anymore, but I still do it in my free time. ;-)

In 2021 I discovered AI through the Cognitive Services when I developed a Windows app as a PoC to extract text from images (which was no longer necessary at the end of the year 2021 or so, because the PowerToys delivered a feature like that ;-). But I kept playing further with the idea and implemented a web application to enable the use of various AI services without Azure or OpenAI account (in the browser and programmatically via the API). Since there is a single API, different operations (translations, ocr, audio transcription, drawing with dall.e) can now be easily combined from a single endpoint. 😃 My best combination so far is creating images with dall.e from audio (in any language supported by Azure ;-) through three layers of AI - audio transcription, translation (to dall.e prompt) and img generation.  

I really like how accessible and easy to use the Azure Cognitive Services are.

I'm eager to see where #30DaysOfAzureAI would take us / me.

**Comment by yihjie** [comment]
Hello Everyone,

I'm YihJie, a presales for cloud integration and support for my customers in Taiwan.
I'll write some sample scripting to demo the Azure solution to the customers but don't write the software or programs to them.
I'll study the AI for my personal interesting and think what it can improve my life.
Let's make the world better with AI ;-)

**Reply by gloveboxes** [reply]
Hey YihJie, lovely to hear from you and hello from Sydney, we're almost in the same time zone 🎉 Great that you are studying AI ❤️

**Reply by arafattehsin** [reply]
Hello @yihjie, just echoing what Dave said. I am also based in Sydney. So good to see you from Taiwan. I am sure you're creating some cool projects with your studies.

**Comment by StephenCleary** [comment]
Hello, all!

I'm Stephen Cleary, a software developer in Michigan in the USA. I'm an MVP but not in the AI space - in fact, I've mostly ignored AI until very recently. I'm now learning how to use it and having lots of fun! It's great that it's supported by .NET - I haven't done Python since the '90s! And I love how the OpenAI APIs make it easy to use - not a data scientist over here.
